src/components/TransactionList.vue

```diff
@@ -61,15 +61,17 @@
                     >
                         <a slot="cashlink" href="#cashlink" @click.prevent="onCreateCashlink">{{ $t('Cashlink') }}</a>
                     </i18n>
-                    <router-link to="buy" class="nq-button light-blue">
-                        {{ $t('Buy NIM') }}
+                    <router-link to="buy">
+                        <button class="nq-button light-blue">
```

Comment:
At some points I need to add `disable` attribute because it changes the UI as well. In our design system, `a:href` elements don't have a `disabled` option, therefore I have encapsulate the button in the `a` element

Reply:
Yeah, I think makes more sense adding `a:disabled` in the style framework, should I do a PR?